setup.py

```diff
@@ -74,21 +74,25 @@ def get_version(version_file):
         # Specify the Python versions you support here. In particular, ensure
         # that you indicate whether you support Python 2, Python 3 or both.
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
     ],
-    python_requires=">=3.6",
+    python_requires=">=3.9",
     # What does your project relate to?
-    keywords="skeleton",
+    keywords="aws ssm ssh",
     packages=find_packages(where="src"),
     package_dir={"": "src"},
-    package_data={"example": ["data/*.txt"]},
     py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
     include_package_data=True,
-    install_requires=["docopt", "schema", "setuptools >= 24.2.0"],
+    install_requires=[
+        "boto3",
+        "boto3-stubs",
+        "docopt",
+        "schema",
+        "setuptools >= 24.2.0",
+        "types-docopt",
+        "types-requests",
+    ],
```

Comment:
i missed this on my first pass-through. Any missing stubs should be added to the `.pre-commit-config.yaml` file instead. They are not required for package functionality which are the only package requirements that should reside in `install_requires`. if we want to include these in more than just the pre-commit configuration then they should go in the `extras_requires["test"]` requirements, a new `extras_requires["dev"]` list, or maybe the `requirements-dev.txt` or `requirements-test.txt` file.

```suggestion
    install_requires=[
        "boto3",
        "docopt",
        "schema",
        "setuptools >= 24.2.0",
    ],
```

Updated `.pre-commit-config.yaml` configuration for `mypy`:

```yaml
        additional_dependencies:
          - boto3-stubs
          - types-docopt
          - types-requests
          - types-setuptools
```

Reply:
I like the idea of adding these in to `extras_requires["test"]` - that will cover those of us who use `setup-env`, though it doesn't resolve the issue (discussed [here](https://github.com/cisagov/awssh/pull/9#discussion_r826380515)) of how to handle missing imports for modules that are missing stubs (like `schema`).

Reply:
@mcdonnnj and @jsf9k- are you good with these type stub dependencies going into the `extras_requires["test"]` section?

Reply:
This has been taken care of in https://github.com/cisagov/awssh/pull/9/commits/3db862992bf8b761fa529989890fe31f7f785c72.